chore(release-notes) updates for 1.2.1 helm... (#339)
redhat-developer/red-hat-developers-documentation-rhdh: modules/release-notes/con-relnotes-fixed-issues.adoc

//link:{LinkRHIDPIssue}RHIDP-1480[RHIDP-1480]
--

Helm deployment shows empty white screen and 404 errors loading static content::
+
--

> A recent change to the upstream Helm chart accidentally prevented deployment of static resources. This is fixed in the 1.2.1 version of the Helm chart.

Reviewer: Maybe use the "With this release," that we always use:

```suggestion
A recent change to the Helm chart accidentally prevented deployment of static resources.

With this release, containing the 1.2.1 version of the Helm chart, installing with Helm deploys the required static resources.
```

Author: ```suggestion
A recent change to the upstream Helm chart accidentally prevented deployment of static resources. 

With the release of the 1.2.1 Helm chart, this is fixed.
```
